File: src/gestorAplicacion/clasesPrincipales/Pasajero.java
package gestorAplicacion.clasesPrincipales;

import java.io.Serializable;
import java.util.ArrayList;

public class Pasajero  implements Serializable  {
	
	private static final long serialVersionUID = 4L;
	
	private String nombre;
	private String apellido;
	private Integer cedula;
	private int edad;
	private String correo;
	private static ArrayList<Pasajero> pasajeros = new ArrayList<Pasajero>();
	
	public Pasajero(String nombre, String apellido, Integer cedula, int edad, String correo) {
		this.nombre = nombre;
		this.apellido = apellido;
		this.cedula = cedula;
		this.edad = edad;
		this.correo = correo;
		Pasajero.pasajeros.add(this);
	}
	
	// -----GETTERS AND SETTERS----- 
	public String getNombre() {
		return nombre;
	}
	public void setNombre(String nombre) {
		this.nombre = nombre;
	}
	public String getApellido() {
		return apellido;
	}
	public void setApellido(String apellido) {
		this.apellido = apellido;
	}
	public Integer getCedula() {
		return cedula;
	}
	public void setCedula(Integer cedula) {
		this.cedula = cedula;
	}
	public int getEdad() {
		return edad;
	}
	public void setEdad(int edad) {
		this.edad = edad;
	}
	public String getCorreo() {
		return correo;
	}
	public void setCorreo(String correo) {
		this.correo = correo;
	}
	public static ArrayList<Pasajero> getPasajeros() {
		return pasajeros;
	}
	public static void setPasajeros(ArrayList<Pasajero> pasajeros) {
		Pasajero.pasajeros = pasajeros;
	} 
	
	
}

File: Python/gestor_aplicacion/clasesPrincipales/Pasajero.py
class Pasajero:
    _pasajeros = []
    def __init__(self, nombre, apellido, cedula, edad, correo):
        #instance fields found by Java to Python Converter:
        self._nombre = nombre
        self._apellido = apellido
        self._cedula = cedula
        self._edad = edad
        self._correo = correo
        Pasajero._pasajeros.append(self)
    # -----GETTERS AND SETTERS----- 
    def getNombre(self):
        return self._nombre
    def setNombre(self, nombre):
        self._nombre = nombre
    def getApellido(self):
        return self._apellido
    def setApellido(self, apellido):
        self._apellido = apellido
    def getCedula(self):
        return self._cedula
    def setCedula(self, cedula):
        self._cedula = cedula
    def getEdad(self):
        return self._edad
    def setEdad(self, edad):
        self._edad = edad
    def getCorreo(self):
        return self._correo
    def setCorreo(self, correo):
        self._correo = correo
    @classmethod
    def getPasajeros(cls):
        return Pasajero._pasajeros

    def __str__(self) -> str:
        return "Nombre del pasajero: {}\nNo Documento: {} ".format(
            self._nombre+self._apellido, self._cedula
        )
